pkg/sdk/comments_test.go

```diff
@@ -0,0 +1,49 @@
+package sdk
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestComments(t *testing.T) {
```

Comment:
We are starting to get more and more functions. Wondering if we should put these into a separate package to make it more clear what these are or if it is sufficient to call them "XXXFunctions". If the latter, then perhaps this should be in the "GeneralDDLFunctions" file. Seems a bit much to make it its own high level thing, but also it is kinda unique. We can discuss this